config.lua

```diff
@@ -0,0 +1,302 @@
+Config = {}
+
+Config.useapartments = true -- false to disable apartments
+Config.useproperties = true -- false to disable properties
+
+-- Minimum price for a property (Avoid Players making their own properties for 1$ for example, i guess?)
+Config.MinimumPrice = 1000
+Config.GardenAndPoolTaxes = true
+Config.Taxes = { -- % of the property price (if price is 1000$ and tax is 10%, then the player pays 1100$), applies to both rent (repeatedly) and buy (once)
+    Garden = 3,
+    Pool = 5,
+    General = 10,
+}

```

Comment:
Am I correct that having three taxes is equivalent to a single tax of 18%? If so, why not just have one config value for property tax?

Reply:
Garden & Pool are toggleable on a property basis (apartments (as properties) don't have a garden for example)
I don't really know if it's a "good" feature or not tbh, seemed simple enough & neat to add so i did.

Reply:
I think the backend should just have one tax rate. If the frontend has multiple tax rates as  a player facing feature that doesn't need to be stored as such in the backend. Have you considered storing tax rate in the database instead? That way you can just go dynamic all the way and individual properties can have their tax rate adjusted for whatever is specific to that property.

Reply:
Maybe not store the tax rate in the database, but i think reworking how i manage the price to allow the tax to be changed at a later date could be nice, like store what taxes have been applied and when the price is actually needed apply those using the configured values, would prevent the problem of creating a property with 18% tax and then the tax is lowered to 15% but the property still costs the same.

Reply:
So now, the Config.Taxes is used with the key as a name and the value as the % of tax to add, when creating a property, a list of those taxes (excluding general) is created for the realtor to select,
the selected taxes are then stored in the property (db), and later used to calculate the price by checking the configured value. (so if you change the value after it was created it will automatically change)